package/yast2-network.changes

```diff
@@ -10437,3 +10437,4 @@ Thu Jun  8 20:46:57 MEST 2000 - dan@suse.cz
 Fri May 19 09:44:08 CEST 2000 - kkaempf@suse.de
 
 - initial version
+
```

Comment:
Apparently I removed it by error and do not want to modify previoys changelog entries :P